definitions/procedures/candlepin/delete_orphaned_records_from_env_content.rb

```diff
@@ -0,0 +1,34 @@
+module Procedures::Candlepin
+  class DeleteOrphanedRecordsFromEnvContent < ForemanMaintain::Procedure
+    metadata do
+      label :cp_env_content_delete_orphaned_records
```

Comment:
Since we have `candlepin` prefix in the candlepin checks, I would recommend following this convention also here. Ideally so that the label matches the class name `:candlepin_delete_orphaned_records_from_env_content`